template/v2/Dockerfile

```diff
@@ -111,6 +111,9 @@ RUN if [[ -z $ARG_BASED_ENV_IN_FILENAME ]] ; \
     micromamba create -n sagemaker-recovery-mode && \
     micromamba install -n sagemaker-recovery-mode -y $JUPYTERLAB_VERSION $SAGEMAKER_JUPYTERLAB_VERSION $SUPERVISOR_VERSION && \
     micromamba clean --all --yes --force-pkgs-dirs && \
+    rm -rf /opt/conda/etc/conda/test-files/ && \
+    find /opt/conda -path "*/site-packages/*/tests" -type d -exec rm -rf {} + 2>/dev/null || true && \
```

Comment:
my only concern is some internal packages include their test files and run them as part of unit test e.g. https://github.com/aws/sagemaker-distribution/blob/main/test/test_artifacts/v2/scripts/run_sagemaker_studio_cli_test.sh